packages/client/package.json

```diff
@@ -20,7 +20,8 @@
     "socket.io-client": "^3.0.3",
     "theme-ui": "^0.3.4",
     "typescript": "^4.1.2",
-    "web-vitals": "^0.2.4"
+    "web-vitals": "^0.2.4",
```

Comment:
What is this for?

Reply:
no idea some built in feature in CRA, it's something about analytics let's leave it there